### DISCUSSION TITLE
how to set date value in normal input and type is date.

### DISCUSSION THREAD

**Discussion by dchirag005** [react-hook-form/react-hook-form#8763] [Q&A] [answered]
Hello, Everyone 
i can facing error in date value not set in normal input field and ffor set value in all field using reset() but only date input value not set. 
(https://user-images.githubusercontent.com/89446061/181758180-8ec15668-ea01-45a2-8465-61b13e0732a3.png)

Set field Code: 
`const obj = {
                firstname: res.data.firstname,
                lastname: res.data.lastname,
                email: res.data.email,
                phoneno: res.data.phoneno,
                department: res.data.department,
                gender: res.data.gender,
                hobbies: res.data.hobbies,
            }
            obj.joindate = moment(res.data.joindate).format('dd/mm/yyyy');
            reset(obj)`

**Comment by Moshyfawn** [comment, marked as answer]
Related? #7162

**Reply by dchirag005** [reply]
Thank you
